vite.config.ts

```diff
@@ -42,16 +42,17 @@ export default defineConfig({
     },
   },
   plugins: [
-    dts({ insertTypesEntry: true }), react(),
+    dts({ insertTypesEntry: true }),
+    react(),
 
-    watchAndRun([
-      {
-        name: 'rebuild',
-        watchKind: ['add', 'change', 'unlink'],
-        watch: path.resolve('src/**/*.(ts|tsx)'),
-        run: 'npm run build',
-        delay: 100,
-      }
-    ])
+    //   watchAndRun([
+    //     {
+    //       name: 'rebuild',
+    //       watchKind: ['add', 'change', 'unlink'],
+    //       watch: path.resolve('src/**/*.(ts|tsx)'),
+    //       run: 'npm run build',
+    //       delay: 100,
+    //     }
+    //   ])
```

Comment:
Observation: Is this being left here for a reason?

Reply:
Ah, yes Alecs you can delete this I guess

Reply:
I forgot to undo it, I commented it out because it slows down my machine when I'm working.

Reply:
I will undo it in the next commit so the approvals don't disappear